addon/services/i18n.js

```diff
@@ -69,7 +69,13 @@ export default Parent.extend(Evented, {
 
   // @private
   _initDefaults: on('init', function() {
-    const ENV = getOwner(this)._lookupFactory('config:environment');
+    let owner = getOwner(this);
+    let ENV;
+    if (owner.factoryFor) {
+      ENV = owner.factoryFor('config:environment').class;
```

Comment:
In general, we should return the object returned by factoryFor (not the .class property)